build-binutils.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+# Description: Builds a standalone copy of binutils
+
+import argparse
+import multiprocessing
+import os
+import pathlib
+import platform
+import shutil
+import subprocess
+import utils
+
+
+def host_arch_target():
+    """
+    Converts the host architecture to the first part of a target triple
+    :return: Target host
+    """
+    host_mapping = {
+        "armv7l": "arm",
+        "ppc64le": "powerpc64le",
+        "ppc": "powerpc"
+    }
+    machine = platform.machine()
+    if machine in host_mapping.keys():
+        return host_mapping[machine]
+    else:
+        return machine
```

Comment:
One more nitpick, this can be replaced with the one-liner 

`return host_mapping.get(machine, machine)` 


> get(key[, default])
> Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a KeyError.